src/Controller/PageController.php

```diff
@@ -61,16 +61,16 @@ public function exceptionsList(): Response
     /**
      * @throws InternalErrorException|AccessDeniedException
      */
-    public function exceptionEdit(Request $request, string $code): Response
+    public function exceptionEdit(Request $request, int $code): Response
```

Comment:
I added this typehint in a previous PR, so no change should be needed on the UPGRADE note.